### PR TITLE
chore: Bump dependencies

### PR DESCRIPTION
**Pull request by o0Ignition0o**
chore: Bump dependencies
- cargo update to bump headers to 0.3.5, thus eliminating the time crate, which is targeted by RUSTSEC-2020-0071
- use test_log in favor of the depreated test_env_log

### CODE REVIEW

File: crates/apollo-router-core/Cargo.toml [hunk 1]
```diff
@@ -25,6 +25,7 @@ once_cell = "1.8.0"
 router-bridge = { git = "https://github.com/apollographql/federation.git", rev = "3aa8f3a533f19e31ab984c87a0674ec78c42ebb6" }
 serde = { version = "1.0.130", features = ["derive", "rc"] }
 serde_json = { version = "1.0.71", features = ["preserve_order"] }
+test-log = { version = "0.2.8", default-features = false, features = ["trace"] }
```
Comment (Geal) [inline crates/apollo-router-core/Cargo.toml:28]: Maybe it should be in dev-dependencies?

Comment (o0Ignition0o) [reply crates/apollo-router-core/Cargo.toml:28]: oh indeed, lemme make sure it's not transitively used in apollo-router tests and apply this!

Comment (Geal) [reply crates/apollo-router-core/Cargo.toml:28]: Also there's test-env-log to remove then on line 40

Comment (o0Ignition0o) [reply crates/apollo-router-core/Cargo.toml]: it passed!

**Review by garypen — APPROVED** [review]
Looks good. Nice work!

**Comment by Geal** [comment]
Yay, less dependencies !

**Review by Geal — APPROVED** [review]

**Comment by cecton** [comment]
For reference about test-log vs test-env-log: https://github.com/d-e-s-o/test-log/issues/13

**Pull request merged**